Dizzy/Screens/Login/AppInfoView.swift

```diff
@@ -0,0 +1,139 @@
+    let privacyPolicySeparatorView: UIView = UIView()
+
+    let contactUsButton: UIButton = UIButton()
+    
+    init() {
+        super.init(frame: CGRect.zero)
+        addSubviews()
+        layoutViews()
+        setupViews()
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    private func addSubviews() {
+        
+        stackView.addArrangedSubview(aboutButton)
+        stackView.addArrangedSubview(aboutSeparatorView)
+        stackView.addArrangedSubview(termsOfUseButton)
+        stackView.addArrangedSubview(termsOfUseSeparatorView)
+        stackView.addArrangedSubview(privacyPolicyButton)
+        stackView.addArrangedSubview(privacyPolicySeparatorView)
+        stackView.addArrangedSubview(contactUsButton)
+        
+        self.addSubview(stackView)
+    }
+    
+    private func layoutViews() {
+        layoutStackView()
+        layoutAboutSeparatorView()
+        layoutTermsOfUseSeparatorView()
+        layoutPrivacyPolicySeparatorView()
+    }
+    
+    private func layoutStackView() {
+        stackView.snp.makeConstraints { stackView in
+            stackView.top.leading.trailing.bottom.equalToSuperview()
+        }
+    }
+    
+    private func layoutAboutSeparatorView() {
+        aboutSeparatorView.snp.makeConstraints { aboutSeparatorView in
+            aboutSeparatorView.height.equalTo(0.5)
+            aboutSeparatorView.width.equalToSuperview().multipliedBy(0.6)
+        }
+    }
+    
+    private func layoutTermsOfUseSeparatorView() {
+        termsOfUseSeparatorView.snp.makeConstraints { termsOfUseSeparatorView in
+            termsOfUseSeparatorView.height.equalTo(0.5)
+            termsOfUseSeparatorView.width.equalToSuperview().multipliedBy(0.6)
+        }
+    }
+    private func layoutPrivacyPolicySeparatorView() {
+        privacyPolicySeparatorView.snp.makeConstraints { privacyPolicySeparatorView in
+            privacyPolicySeparatorView.height.equalTo(0.5)
+            privacyPolicySeparatorView.width.equalToSuperview().multipliedBy(0.6)
+        }
+    }
+    private func setupViews() {
+        setupStackView()
+        setupAboutButton()
+        setupAboutSeparatorView()
+        setupTermsOfUseButton()
+        setupTermsOfUseSeparatorView()
+        setupPrivacyPolicyButton()
+        setupPrivacyPolicySeparatorView()
+        setupContactUsButton()
+    }
+    
+    private func setupStackView() {
+        stackView.axis = .vertical
+        stackView.alignment = .center
+    }
+    
+    private func setupAboutButton() {
+        
+        aboutButton.setTitleColor(.lightGray, for: .normal)
+        aboutButton.titleLabel?.font = Fonts.h3()
+        aboutButton.setTitle("About".localized, for: .normal)
+    }
+    private func setupAboutSeparatorView() {
+        aboutSeparatorView.backgroundColor = UIColor(red: 97/255, green: 97/255, blue: 97/255, alpha: 1.0)
```

Comment:
you need to put the color value as class variable

Reply:
Done!